apps/remix-ide/src/app.js

```diff
@@ -16,7 +16,12 @@ import { HiddenPanel } from './app/components/hidden-panel'
 import { VerticalIcons } from './app/components/vertical-icons'
 import { LandingPage } from './app/ui/landing-page/landing-page'
 import { MainPanel } from './app/components/main-panel'
-import FetchAndCompile from './app/compiler/compiler-sourceVerifier-fetchAndCompile'
+
+import { CompilerArtefacts } from '@remix-core-plugin/compiler-artefacts'
```

Comment:
Perhaps this need to be prefixed with `@remix-project/` instead of `@remix-core-plugin`.
`remix-project` is the name of the org in npm.
If this is just an alias then I guess this is ok
@ioedeveloper @Aniket-Engg  any thoughts?

Reply:
Sure I did it this way because you suggested it but maybe you meant something else.

Reply:
sure, I mean that `@remix-core-plugin` don't rely to any npm organization we own, so I don't see how you can publish that.
I was thinking that this could be something like:
`import { CompilerArtefacts } from '@remix-project/core-plugin'`

Reply:
Ok, so then we bundle all the plugins in core-plugin package?